Win11ThemeTest/TextBoxTest.cs

```diff
@@ -0,0 +1,449 @@
+﻿using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Input;
+using FlaUI.UIA3;
+using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Conditions;
+using FlaUI.Core.Definitions;
+using FlaUI.Core.Input;
+using FlaUI.UIA3;
+using FlaUI.UIA3.Identifiers;
+using FlaUI.UIA3.Patterns;
+using System.Drawing;
+using System.Linq;
+using NuGet.Frameworks;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+using System.Runtime.ConstrainedExecution;
+
+namespace Win11ThemeTest
+{
+    [TestFixture]
+    public class textBoxTests
+    {
+        private Application app;
+        private Window window;      
+        private Window MainWindow;
+        private Window TextWindow;
+        private Window btnWindow;
+        MenuItems menuItemWindow;
+        TextBox textBox;
+        TextBox disablexTextBox;
+        Button txtButton;
+        TextBox multilineTextbox;
+        public textBoxTests()
+        {
+            app = FlaUI.Core.Application.Launch(@"..\\..\\..\\..\\TestingApplication\\bin\\Debug\\net9.0-windows\\win-x64\\TestingApplication.exe");
+            using (var automation = new UIA3Automation())
+            {
+                MainWindow = app.GetMainWindow(automation);
+                txtButton = MainWindow.FindFirstDescendant(cf => cf.ByAutomationId("txtBoxButton")).AsButton();
+                Mouse.Click(txtButton.GetClickablePoint());
+                Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(2000));
+                TextWindow = MainWindow.FindFirstDescendant(cf => cf.ByName("TextWindow")).AsWindow();
+                btnWindow = MainWindow.FindFirstDescendant(cf => cf.ByName("lbl")).AsWindow();               
+            }
+        }
+
+        [Test]
```

Comment:
Consider closing windows after test execution.

Reply:
Added z_cleanup test at last after test execution to close Textwindow and Mainwindow.